bin/is-website-vulnerable.js

```diff
@@ -5,65 +5,70 @@
 const os = require('os')
 const debug = require('debug')('is-website-vulnerable')
 const argv = require('yargs').argv
+const inquirer = require('inquirer')
 const { Audit, RenderConsole, RenderJson, Utils } = require('../index')
 
-let url = process.argv[2]
-debug(`received url argument: ${url}`)
+const main = async () => {
+  let url = process.argv[2]
+  debug(`received url argument: ${url}`)
 
-url = Utils.parseUrl(url)
+  url = Utils.parseUrl(url)
 
-if (!url) {
-  console.error('')
-  console.error('error: please provide a URL of a website to scan')
-  console.error('')
-  console.error('Usage:')
-  console.error('  is-website-vulnerable https://www.google.com')
-  console.error('')
-  process.exit(1)
-}
+  if (!url) {
+    console.error('Woops! You forgot to provide a URL of a website to scan.')
 
-const isWindows = os.type() === 'Windows_NT'
-const isJson = !!argv.json
-const showProgressBar = !isJson && !isWindows
+    const question = {
+      type: 'input',
+      name: 'url',
+      message: 'Please provide a URL here:',
+      validate: input => input && input.length > 0
```

Comment:
Note that we could validate given URLs (both through the prompt & as CLI arguments) in a more verbose way, such as using [`is-url-superb`](https://github.com/sindresorhus/is-url-superb). I didn't add it yet, because I wanted to consult with others and ask what you think about the idea.

Reply:
I think it's a good idea for a follow-up PR ✨
Let's land this one first!